models/account.go

```diff
@@ -2,20 +2,41 @@
 // of the accounts service.
 package models
 
-import "github.com/google/uuid"
+import (
+	"context"
+
+	"github.com/google/uuid"
+)
 
 type Account struct {
-	ID uuid.UUID `json:"id"`
+	ID    uuid.UUID `json:"id" bson:"_id,omitempty"`
+	Email string    `json:"email" bson:"email,omitempty"`
+	Name  string    `json:"name" bson:"name,omitempty"`
+	Hash  *[]byte   `json:"hash" bson:"hash,omitempty"`
+}
```

Comment:
Mhm, I don't know whether we should have `bson` annotations here because it's supposed to be database independent but duplicating each struct would be tedious. Anyway we'll handle that next sprint.